### PR TITLE
Update Journal issn field to type:issn, and add MedAbbr value to nlmta field

### PR DESCRIPTION
**Pull request by jrmartino**
resolves #116

This update encodes the ISSN type (Print, Online) by prepending it, with a : separator, to the ISSN value. This adjustment is made for both the medline and the pmc readers. In addition, we add the MedAbbr value on medline journals to the nlmta field on Journal. Dependencies and context are updated. An IT was updated to follow the intent of the comment in the test.

### CODE REVIEW

File: pass-journal-loader-integration/src/test/java/org/dataconservancy/pass/loader/journal/integration/DepositIT.java [hunk 1]
```diff
@@ -96,14 +96,16 @@ public void loadFromFileTest() throws Exception {
...
+                .withEnv("PASS_FEDORA_BASEURL", PASS_BASEURL)
+                .withEnv("LOG_ORG_DATACONSERVANCY_PASS", "DEBUG")
                 .start();
 
+        wait(load);
+
         // The last dataset removed a type A journal, so now we expect only one
         assertEquals(3, listJournals().size());
-        assertEquals(2, typeA(listJournals()).size());
+        assertEquals(1, typeA(listJournals()).size());
```
Comment (birkland) [inline pass-journal-loader-integration/src/test/java/org/dataconservancy/pass/loader/journal/integration/DepositIT.java:108]: What's the deal here - I don't see a change to the test data that suggests the list of "type A" journals changes.  Then again, I don't fully remember what the CSV has in it.  Can you confirm that the type A stats are correct?

**Review by birkland — COMMENTED** [review]
Looks good, though a question on the "type-A-ness" of the results.  A test changes the count of expected type A journals, but it doesn't look like the test data changed with respect to the number of type A journals?

**Comment by jrmartino** [comment]
Aaron,

     My understanding was that type-A ness was for active journals - 
those which did not have an end date in the csv file. Adding an end date 
in the last field should de-type-a-ify a journal. I think that is the 
behavior that the unit test was verifying. The second journal in the 
second csv file has an added end date, but did not have it in the first.


Jim

**Review by birkland — APPROVED** [review]
I see, looks good

**Comment by htpvu** [comment]
@birkland does that means that this PR can be merged? or is there other things to look into before merging?

**Comment by birkland** [comment]
I think it looks good and can be merged

**Pull request merged**